header/p2p/peer_stats.go

```diff
@@ -40,6 +40,16 @@ func (p *peerStat) updateStats(amount uint64, time uint64) {
 	p.peerScore = (p.peerScore + averageSpeed) / 2
 }
 
+// decreaseScore decreases peerScore by 20% of the peer that failed the request by any reason.
+// NOTE: decreasing peerScore in one session will not affect its position in queue in another
+// session(as we can have multiple sessions running concurrently).
```

Comment:
What is your thinking about 20%?

Reply:
We need some random value. Decided to make it 20%.

Reply:
Issue

Reply:
Pls